server/server.js

```diff
@@ -27,16 +27,16 @@ module.exports = function Server(app, options = {}) {
   }));
   app.post("/notify", notifHandler({
     hostSecret,
-    groupTraits: true,
     onSubscribe() {
       console.warn("Hello new subscriber !");
     },
     handlers: {
-      "user:update": (ctx, messages = []) => {
-        return Promise.all(messages.map(message => updateUser({ message }, {
-          ship: ctx.ship,
-          hull: ctx.client
-        })));
+      "user:update": ({ ship, client: hull }, messages = []) => {
+        return Promise.all(messages.map(message => {
+          // TODO: enable groupTraits option in the notifHandler
```

Comment:
I think you can use the `groupTraits` option but inside `userHandlerOptions` as in this example: https://github.com/hull/hull-node/#notifhandler